elixir/apps/web/lib/web/live/sites/new_token.ex

```diff
@@ -98,26 +98,26 @@ defmodule Web.Sites.NewToken do
               ><%= debian_command_install() %></.code_block>
 
               <p class="p-6">
-                Step 3: Copy the token:
+                Step 3: Configure a token:
               </p>
 
               <.code_block
-                id="code-sample-debian3"
+                id="code-sample-debian4"
                 class="w-full text-xs whitespace-pre-line"
                 phx-no-format
                 phx-update="ignore"
-              ><%= token(@env) %></.code_block>
+              ><%= debian_command_authenticate() %></.code_block>
```

Comment:
Steps 3 and 4 were swapped to the order you need to perform this to not clobber the clipboard.